reset.go

```diff
@@ -0,0 +1,59 @@
+package goose
+
+import (
+	"database/sql"
+	"log"
+	"sort"
+)
+
+// Reset rolls back all migrations
+func Reset(db *sql.DB, dir string) error {
+	migrations, err := CollectMigrations(dir, minVersion, maxVersion)
+	if err != nil {
+		return err
+	}
+	statuses, err := dbMigrationsStatus(db)
```

Comment:
that doesn't work for me. Because we use this command to reset db state between branches sometimes we have:
- migration1 (applied)
- migration2 (not applied)
- migration3 (applied)

if I take current migration it will be m3. Then it will rollback m2 and command will fail.